types/src/bloom.rs

```diff
@@ -1,4 +1,21 @@
-use alloy_rlp::RlpEncodableWrapper;
+use alloy_rlp::Encodable;
 
-#[derive(Debug, RlpEncodableWrapper, PartialEq, Clone)]
-pub struct Bloom(pub [u8; 256]);
+use crate::H160;
+#[cfg_attr(feature = "std", derive(serde::Serialize, serde::Deserialize))]
```

Comment:
Don't we need this to work for `no_std` too...? CC @baidang201

Reply:
@xcthulhu yes, need it for no_std

Reply:
Resolved by @baidang201